pkgs/by-name/go/go-pray/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  pkg-config,
+  alsa-lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "go-pray";
+  version = "0.1.13";
+
+  src = fetchFromGitHub {
+    owner = "0xzer0x";
+    repo = "go-pray";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-wi/sUjzNbVLbqwtM1BglmXgkaqMamjIsGS6EjlyeC9Y=";
+  };
+  vendorHash = "sha256-qMTg2Vsk0nte1O8sbNWN5CCCpgpWLvcb2RuGMoEngYE=";
+
+  nativeBuildInputs = [
+    pkg-config
+    alsa-lib
+  ];
+  ldflags = [
+    "-X 'github.com/0xzer0x/go-pray/internal/version.version=${finalAttrs.version}'"
+    "-X 'github.com/0xzer0x/go-pray/internal/version.buildCommit=${finalAttrs.version}'"
```

Comment:
Is this useful? We're setting the commit to a version string. You can actually get the commit hash, but I wouldn't recommend it, it's a bit messy.